data/json/items/armor/suits_protection.json

```diff
@@ -594,6 +594,26 @@
     "flags": [ "VARSIZE", "STURDY" ],
     "armor": [ { "encumbrance": 15, "coverage": 80, "covers": [ "torso", "arm_l", "arm_r", "leg_l", "leg_r" ] } ]
   },
+   {
+    "id": "k_gambeson",
+    "type": "ARMOR",
+    "name": { "str": "gambeson" },
+    "description": "A thick jacket of quilted fabric, designed to be worn underneath mail or other armor. This one incorporates kevlar layers added for better protection.",
+	'weight": "453 g"
```

Comment:
It should have a weight based on the amount of kevlar used, not the weight of a regular gambeson

Reply:
Raised the cost up to 36 layered kevlar panels, that gives a weight of 2.880 kg and 2.52 L, which is close to the gambeson's 3 L. 

 It looks like the gambeson itself is off however, it's 453 g, which doesn't add up based on the material, 26 rags at 80 g should be 2080 g.